vocabularies/Common.json

```diff
@@ -1289,6 +1289,88 @@
       "@Common.Experimental": true,
       "@Core.IsURL": true,
       "@Core.Description": "Base URL for WebSocket connections"
+    },
+    "RecommendationsFunction": {
+      "$Kind": "Term",
+      "$Type": "Common.QualifiedName",
+      "$AppliesTo": ["EntityType"],
+      "@Common.Experimental": true,
+      "@Core.Description": "Qualified name of a function that computes AI-based recommendations for entities of the annotated type (see [this template](#Template_RecommendationsFunction))"
+    },
+    "Template_RecommendationsFunction": [
+      {
+        "$Kind": "Function",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.Description": "Template for functions that compute AI-based recommendations for the bound entity and its related entities",
+        "@Core.LongDescription": "The function SHOULD compute the recommendations asynchronously, see [this diagram](../docs/recommendations.md).\n\nThe template function itself cannot be invoked.",
+        "$Parameter": [
+          { "$Name": "Entity", "$Type": "Edm.EntityType" },
+          {
+            "$Name": "Targets",
+            "$Collection": true,
```

Comment:
@HeikoTheissen, I guess we discussed in the past that complex types can't be passed as input parameters to OData Function. You recommended in that case to stringify and sent it. Today while looking at the PR we realized that we still have a collection here and RAP can't support this. Can we change this?

Reply:
Change it to what? We need a collection of target URLs, how else to represent this than as a collection of strings? A whitespace-separated list of strings?

Reply:
> complex types can't be passed as input parameters to OData Function

Which SAP tool introduces this restriction?

Reply:
SADL, see https://jira.tools.sap/browse/FIORITECHP1-22187.

Reply:
Looking into it...